lib/Reader/readerir.cpp

```diff
@@ -7462,11 +7462,15 @@ void GenIR::addPHIOperand(PHINode *Phi, Value *NewOperand,
     LLVMBuilder->restoreIP(SavedInsertPoint);
   }
 
-  int BlockIndex = Phi->getBasicBlockIndex(NewBlock);
-  if (BlockIndex >= 0)
-    Phi->setIncomingValue(BlockIndex, NewOperand);
-  else
-    Phi->addIncoming(NewOperand, NewBlock);
+  bool FoundBlockOperand = false;
```

Comment:
No warning in release builds probably because the var is not only initialized but re-assigned later. We do have several warnings about other initialized but not referenced variables in release builds. I'll fix those separately.